Ruby-on-Rails-application/sQuiz/app/controllers/api/groups_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::GroupsController < ApplicationController
+  
+  acts_as_token_authentication_handler_for Instructor
+
+  def add
```

Comment:
@Youssef-Emad  you have to check if the student was found in the group before or not